glide.yaml

```diff
@@ -1,16 +1,19 @@
 package: github.com/openshift/generic-admission-server
 import:
 - package: k8s.io/code-generator
-  version: kubernetes-1.10.0
+  version: kubernetes-1.11.0
 - package: k8s.io/client-go
-  version: kubernetes-1.10.0
+  version: kubernetes-1.11.0
 - package: k8s.io/apimachinery
-  version: kubernetes-1.10.0
+  version: kubernetes-1.11.0
 - package: k8s.io/apiserver
-  version: kubernetes-1.10.0
+  version: kubernetes-1.11.0
 - package: k8s.io/api
-  version: kubernetes-1.10.0
+  version: kubernetes-1.11.0
 
 # select compatible cobra version for pflags 9ff6c6923cfffbcd502984b8e0c80539a94968b7 used in Kubernetes 1.9
 - package: github.com/spf13/cobra
   version: c439c4fa093711d42e1b01acb1235b52004753c1
+# we need a specific level of jsoniter to match kube
+- package: github.com/json-iterator/go
+  version: f2b4162afba35581b6d4a50d3b8f34e33c144682
```

Comment:
this is not derived from k8s.io/apimachinery?

Reply:
> this is not derived from k8s.io/apimachinery?

Oddly not.  Help diagnosing it would be welcome.  We hit the same thing in openshift/origin, but there I just assumed we had a random dep somewhere.